app/models/message.rb

```diff
@@ -0,0 +1,8 @@
+class Message < ApplicationRecord
+  belongs_to :conversation
+  belongs_to :user
+
+  validates :text, :conversation, :user, presence: true
```

Comment:
Same here, no need to validate presence of `conversation` and `user`

Reply:
Yes, they will be gone in next PR